### PR TITLE
serd: update to 0.30.10.

### PR DESCRIPTION
**Pull request by tibequadorian**
enable tests

#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/serd/template [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
-checksum=f5a2c74c659d8b318059068f135a43a3771491c367b6947e053a713b23cd37ef
+checksum=affa80deec78921f86335e6fc3f18b80aefecf424f6a5755e9f2fa0eb0710edf
+
+if [ "${XBPS_CHECK_PKGS}" ]; then
+	configure_args+=" --test"
+fi
+
+do_check() {
+	./waf test
+}
```
Comment (ericonr) [inline srcpkgs/serd/template:19]: Maybe we can add this to the build style? what do you think?

Comment (tibequadorian) [reply srcpkgs/serd/template:19]: Yes I think too!

Comment (tibequadorian) [reply srcpkgs/serd/template:19]: But I don't know if I'm ready to mess around with the build styles :D

Comment (ericonr) [reply srcpkgs/serd/template:19]: I will leave this as a future improvement, we can check if other waf stuff also has tests.

Comment (tibequadorian) [reply srcpkgs/serd/template:19]: It looks like `waf test` is officially documented, but doesn't always exist. And I didn't find a way to check if a build script contains a test task or not, like with `make -q`

**Pull request merged**